<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsApplication1Class</class>
 <widget class="QWidget" name="QtWidgetsApplication1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>557</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>WaitCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsApplication1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>651</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(230, 213, 47, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>95</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(175, 109, 160, 255), stop:1 rgba(255, 255, 255, 255));
font: 700 11pt &quot;Microsoft YaHei UI&quot;;
font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="import_mesh">
    <property name="cursor">
     <cursorShape>WaitCursor</cursorShape>
    </property>
    <attribute name="title">
     <string>导入网格文件</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Import Mesh</string>
       </property>
       <property name="autoRepeatDelay">
        <number>600</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Sphere Mesh</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget">
    <attribute name="title">
     <string>求解器设置</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="MRTLBM">
       <property name="text">
        <string>MRT-LBM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SRTLBM">
       <property name="text">
        <string>SRT-LBM</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>边界设置</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="Inlet">
       <property name="text">
        <string>Inlet direction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ZhouHe">
       <property name="text">
        <string>Zhou-He</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>物理信息查询</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="Calculate">
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Porosity">
       <property name="text">
        <string>Porosity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Permeability">
       <property name="text">
        <string>Permeability</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="output_mesh">
    <property name="cursor">
     <cursorShape>WaitCursor</cursorShape>
    </property>
    <attribute name="title">
     <string>导出网格/结果文件</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Output Mesh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OutputResult">
       <property name="text">
        <string>Output Result</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>354</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/流线/liuxian.jpg);</string>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>241</width>
     <height>211</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/流场/liuchang.jpg);</string>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>344</y>
     <width>211</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/云图/cloud.jpg);</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgetsApplication1.qrc"/>
  <include location="x64/Debug/picture/picture.qrc"/>
 </resources>
 <connections/>
</ui>
